vacationtime-solution/src/Code.js

```diff
@@ -0,0 +1,186 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Create column number variables.
+var COLUMN_NUMBER = {
+  EMAIL: 2,
+  NAME: 3,
+  MANAGER_EMAIL: 4,
+  START_DATE: 5,
+  END_DATE: 6,
+  APPROVAL: 8,
+  NOTIFIED: 9,
+};
+
+// Global variables.
```

Comment:
Same, this doesn't give the reader any new information so I would remove. The reader will know these are meant to be available globally and they are "constant" variables.